### PR TITLE
Spreadsheet check

### PR DESCRIPTION
**Pull request by kamfonik**
Add a script that can by run by cron to check for new approved requests and notify the helpdesk automatically.

This way all the approving MOC person has to do to is type "approved" in the right spreadsheet column.

WIP because we will go over the notification format on Friday with the helpdesk, some changes may be needed after that meeting.

EDIT: meeting moved to Wednesday 3/22

### CODE REVIEW

**Comment by kamfonik** [comment]
We tested the template with the helpdesk today, it worked.

File: check-approved-requests.py [hunk 1]
```diff
@@ -0,0 +1,229 @@
+#   Copyright 2016 Massachusetts Open Cloud
```
Comment (xuhang57) [inline check-approved-requests.py]: 2017?

File: check-approved-requests.py [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+    # farm it out to a function somewhere to streamline updates.  Possibly
+    # also update to use a list and for i in range().
+    quotas = {'instances': cells[10],
+              'cores': cells[11],
+              'ram': cells[12],
+              'floatingip': cells[13],
+              'volumes': cells[14],
+              'snapshots': cells[15],
+              'gigabytes': cells[16]}
+    unchanged_quotas = [q for q in quotas if quotas[q] == '']
```
Comment (xuhang57) [inline check-approved-requests.py]: I normally would avoid putting variable names as one letter. But in this case, it is clear. Just saying :)

File: check-approved-requests.py [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+                          subject=subject, **request_info)
+    msg.send()
+
+
+def timestamp_spreadsheet(sheet, time, processed_rows):
+    """Mark the given rows as notified in the Google Sheet"""
+    # FIXME: consider whether spreadsheet.py should do some of this work
+    
+    # FIXME: Specify worksheet name in config
+    worksheet = 'Form Responses 1'
```
Comment (xuhang57) [inline check-approved-requests.py]: Oh, I see.

File: check-approved-requests.py (hunk 2 above)
Comment (kamfonik) [inline check-approved-requests.py]: Yeah, it's generally a terrible idea, but I think in this specific case it's kind of like having an iterator variable, so it's OK.  

I think at one point I had it as `quota for quota in quotas if quotas[quota] = ''` but that was really hard to read.

File: check-approved-requests.py (hunk 1 above)
Comment (kamfonik) [inline check-approved-requests.py]: good catch, fixed

File: check-approved-requests.py (hunk 2 above)
Comment (xuhang57) [inline check-approved-requests.py]: That's true and it would violate a certain pep8 rule since it is too long I guess.

**Pull request merged**